### DISCUSSION TITLE
OS icons

### DISCUSSION THREAD

**Discussion by ruadoVn** [JanDeDobbeleer/oh-my-posh#1522] [Troubleshoot]
How can i change the default icons of OS?

**Comment by JanDeDobbeleer** [comment]
The OS segment specifies [properties](https://ohmyposh.dev/docs/os#properties) per OS.

**Comment by svenlowry-cgi** [comment]
Is there a way to get the OS icon to change based on the OS the shell is running in?
Currently I have `"template": "{{if .Root}} \uF0E7 \uE62A {{else}} \uE62A {{end}}"` in my Windows profile and `"template": "{{if .Root}} \uF0E7 \uF31b {{else}} \uF31b {{end}}"` in my WSL (Ubuntu) profile.
Can I replace the `\uE62A` and `\uF31b` with something so I can use the same profile file in each setup, and have it load the Windows icon in PowerShell and Ubuntu icon in WSL?

**Reply by JanDeDobbeleer** [reply]
You can use the OS segment that has this ability.

**Reply by svenlowry-cgi** [reply]
Ah, perfect, I just changed my segment type from "session" to "os" and it's done exactly what I wanted. Thank you!
